rskj-core/src/main/java/org/ethereum/net/server/ChannelManagerImpl.java

```diff
@@ -96,9 +103,28 @@ public void start() {
 
     @Override
     public void stop() {
+        saveActivePeersToFile();
```

Comment:
saving/loading of last discovered peers should probably be tackled in the `PeerExplorer` class. `ChannelManagerImpl` deals with the wire protocol, not discovery, and may include a subset of discovered peers

Reply:
Updated!